docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,220 @@
+
+
+### IPMX 
+  The IPMX (Internet Protocol for Media Experience) is a set of open standards and specifications to enable the carriage of compressed and uncompressed video, audio and data over IP networks. It includes provisions for control, copy protection, connection management and security. Includes use of SMPTE and NMOS standards.
+
+
+### IPMX Controller
+  An IPMX Controller is a type of NMOS Controller analogous to an NMOS Broadcast Controller. The IPMX Controller is at the application layer of the NMOS Architecuture and makes use of IPMX APIs including AMWA NMOS APIs. Further information can be found in [EDID-CM-ARCH](#edid-cm-arch-ref).  
+
+### EDID 
+  Extended Display Identification Data (EDID) is a metadata format for display devices to describe their capabilities to a video source (e.g. graphics card or set-top box). The data format is defined by a standard published by the Video Electronics Standards Association (VESA).  For purposes of this BCP, EDID definitions are taken from the E-EDID version 1.4 [E-EDID](#edid-ref).
+
+## Mapping from EDID to NMOS Resources
+
+This section is broken down into several subsections that describe recommendations for mapping parts of a target display device's EDID to NMOS Resources.  The following subsections are provided:
+
+- Mapping of EDID to IS-04 Receiver Caps 
+   - Video Mappings from EDID to IS-04 Receiver Caps
+   - Audio Mappings from EDID to IS-04 Receiver Caps
+- Mapping of EDID to IS-EDID-CM
+   - EDID to IS-EDID-CM Destination Resource
+   - EDID to Essential Media Capabilities
+   - EDID to Raw EDID Format
+
+### Constuction of IS-04 Receiver Caps
+
+This section describes how a display device's capabilities (video and audio) as described by its EDID is mapped by an implementation to the NMOS device's [IS-04](#is-04-ref) Receiver Capabilities.  
+
+#### Video and AUdio Mappings
+
+The following refers to the NMOS Receiver Capabilities Best Current Practices [BCP-004-01](#bcp-004-01-ref). Paremeter values used are taken from the NMOS Capabilities Parameter Register [NMOS-CAP-REG](#nmos-cap-reg-ref)
+
+##### Video Mappings
+
+In order that a NMOS device that provides IS-04 Receivers can work with EDID type video devices the device:
+
+- MUST create one IS-04 Video Receiver for each of the displays associated with the IS-04 Device
+
+- MUST set the `format` element to the value `urn:x-nmos:format:video`  
+
+- MUST associate with each IS-04 Receiver a `contraints_set` resource in the IS-04 Receiver's `caps` resource  as described in [BCP-004-1]
+
+- MUST map at least one of the video modes present by the display device's EDID into the created `contraint_set` with the following elements set:
+
+  - `urn:x-nmos:cap:meta:enabled` set to `true`.  
+
+  - `urn:x-nmos:cap:format:grain_rate` set to the refresh rate as taken from the EDID's preferred timing mode for the supported video format. 
```

Comment:
What "bits" of the EDID does this refer to? My impression is that there's a lot of ambiguity perhaps we need to be very specific?